howto/spark/spark_operator.ipynb

```diff
@@ -0,0 +1,943 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Spark Job with Spark Operator\n",
+    "Using spark operator for running spark job over k8s."
```

Comment:
Please add some documentation regarding:
- What is the spark operator
- Pros / Cons (When should i use?) about the "serverless" functionality and the image requirements
- How the process works: MLRun -> Yaml to spark operator -> resource allocation -> running ...